docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Standard models"
+sidebar_position: 2
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+### Overview
+Standard data models are authored and maintained by Snowplow. Follow below steps to create one.
```

Comment:
```suggestion
Standard data models are authored and maintained by Snowplow. Follow below the steps below to create and run one.
```

Reply:
I'm going to remove extra "below" => "Follow the steps below to create and run one."